test/extended/util/framework.go

```diff
@@ -1018,87 +1017,88 @@ func CheckBuildCancelled(b *buildv1.Build) bool {
 	return b.Status.Phase == buildv1.BuildPhaseCancelled
 }
 
+func getServiceAccount(c corev1client.ServiceAccountInterface, name string) (*corev1.ServiceAccount, error) {
+	sc, err := c.Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		if kapierrs.IsNotFound(err) || kapierrs.IsForbidden(err) {
+			return nil, nil
```

Comment:
This is a blocker.  We don't do this when working with kube APIs because it defies expectations about client handling that have existed for the past eight years.

Reply:
Left it as it was before.